v3/lints/rfc/lint_subject_given_name_max_length.go

```diff
@@ -52,11 +69,11 @@ func (l *subjectGivenNameMaxLength) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *subjectGivenNameMaxLength) Execute(c *x509.Certificate) *lint.LintResult {
-	for _, j := range c.Subject.GivenName {
-		if utf8.RuneCountInString(j) > 16 {
+	for _, givenName := range c.Subject.GivenName {
+		characters := utf8.RuneCountInString(givenName)
+		if characters > 32768 {
```

Comment:
This passes if characters is exactly 32768, but the lint description says "MUST be less than 32768 characters". Which is right?

Reply:
That's a good point and is dependent on whether notation for `(SIZE (1..ub-name))` is inclusive or not. I actually tried to find wording last weekend on whether it was or not, but I will have to dig deeper as I couldn't find it.